shared/src/map/MapScene.cpp

```diff
@@ -244,3 +248,52 @@ void MapScene::pause() {
             }
         }));
 }
+
+void MapScene::drawReadyFrame(const ::RectCoord & bounds, float timeout, const std::shared_ptr<MapReadyCallbackInterface> & callbacks) {
+
+    for (const auto &layer : layers) {
+        layer.second->enableAnimations(false);
```

Comment:
maybe we should re-enable the animation after the drawing?

Reply:
XX